drivers/xen/xengnttab/Makefile.uk

```diff
@@ -0,0 +1,7 @@
+$(eval $(call addlib_s,libxengnttab,$(CONFIG_LIBXENGNTTAB)))
+
+CINCLUDES-$(CONFIG_LIBXENGNTTAB)        += -I$(LIBXENGNTTAB_BASE)/include
+
+LIBXENGNTTAB_SRCS-y                     += $(LIBXENGNTTAB_BASE)/gnttab.c
+LIBXENGNTTAB_SRCS-$(CONFIG_ARCH_X86_64) += $(LIBXENGNTTAB_BASE)/x86/gnttab.c|x86
```

Comment:
Unless we have a different convention on Xen, under the current conventions these should be under `$(LIBXENGNTTAB_BASE)/arch/arm64/` and `$(LIBXENGNTTAB_BASE)/arch/x86_64/`

Reply:
Sure, I will adopt it. I kept the original structure during the moving and currently think that we might be too strict with our convention. But you are right, we should stick to it and update the convention first.